parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -466,8 +466,32 @@ configGPUDrivers() {
     retrycmd_if_failure 120 5 25 $GPU_DEST/bin/nvidia-smi || exit $ERR_GPU_DRIVERS_START_FAIL
     retrycmd_if_failure 120 5 25 ldconfig || exit $ERR_GPU_DRIVERS_START_FAIL
 }
+
+validateGPUDrivers() {
+    {{/* will install GPU drivers if not loaded correctly */}}
```

Comment:
nit: remove this to minimize size of the cse payload as it's delivered through cloud-init that has a size limit of 64K